libimage/filters.go

```diff
@@ -50,13 +58,14 @@ func (r *Runtime) filterImages(ctx context.Context, images []*Image, options *Li
 		return nil, err
 	}
 	result := []*Image{}
-	for _, i := range images {
-		match, err := matches(i, filters)
+	for i := range images {
+		image := images[i]
+		match, err := image.applyFilters(filters)
```

Comment:
Can we do `images[i].applyFilters` here? then we won't need the extra `image` var above

Reply:
sure